setup.py

```diff
@@ -13,31 +15,24 @@
     cytominer-database=cytominer_database.command:command
     """,
     long_description="cytominer-database provides mechanisms to import CSV "
-                     "files generated in a morphological profiling experiment "
-                     "into a database backend. "
-                     "Please refer to the online documentation at "
-                     "http://cytominer-database.readthedocs.io",
+    "files generated in a morphological profiling experiment "
+    "into a database backend. "
+    "Please refer to the online documentation at "
+    "http://cytominer-database.readthedocs.io",
     package_data={
-        'cytominer_database': [
-            'config/*.ini',
-            'config/*.json',
-            'config/*.sql'
-        ],
+        "cytominer_database": ["config/*.ini", "config/*.json", "config/*.sql"]
     },
-    packages=setuptools.find_packages(
-        exclude=[
-            'tests',
-            'doc'
-        ]
-    ),
+    packages=setuptools.find_packages(exclude=["tests", "doc"]),
     include_package_data=True,
     install_requires=[
-        'backports.tempfile>=1.0rc1',
-        'click>=6.7',
-        'configparser>=3.5.0',
-        'csvkit>=1.0.2',
-        'pandas>=0.20.3'
+        "backports.tempfile>=1.0rc1",
+        "click>=6.7",
+        "configparser>=3.5.0",
+        "csvkit>=1.0.2",
+        "pandas>=0.20.3",
+        "sphinx>=2.2.2",
```

Comment:
@gwaygenomics we don't need `sphinx` and `sphinx-rtd-theme` for install, correct?

Reply:
I was able to install, test, and also build the docs even without these additions to `setup.py`

Reply:
correct. This is leftover from a previous option to autobuild. I will remove!